app/src/main/java/fr/free/nrw/commons/utils/ViewUtil.java

```diff
@@ -85,14 +83,16 @@ public static void hideKeyboard(View view){
      * A snack bar which has an action button which on click dismisses the snackbar and invokes the
      * listener passed
      */
-    public static void showDismissibleSnackBar(View view, int messageResourceId,
-        int actionButtonResourceId, View.OnClickListener onClickListener) {
+    public static void showSnackBar(View view,
```

Comment:
What's the rationale behind renaming this method?

Reply:
Have undone this change. :)